README.md

```diff
@@ -186,7 +228,7 @@ pip install -e .
 staramr 
 ```
 
-Due to the way I package the ResFinder/PointFinder databases, the development code will not come with a default database.  You must first build the database before usage. E.g.
+Due to the way I package the ResFinder/PointFinder/PlasmidFinder databases, the development code will not come with a default database.  You must first build the database before usage. E.g.
```

Comment:
You can change **I** to **we** since there's more than one of us writing the software now :smile:

Reply:
Solved in commit 2202e45